@utahdts/utah-design-system/react/components/forms/FileInput/context/FileContext.js

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react';
```

Comment:
FileContext not needed.

Reply:
This entire file is not needed - Joe translation